docs/develop/customize.md

```diff
@@ -64,3 +64,26 @@ export function ResultsItemTemplate(record, index) {
 ```
 
 Then, run the `invenio-cli update --no-install-js` command as above and refresh the page! You should be able to see more text in each result's description! You can find all the available templates [here](https://github.com/inveniosoftware/invenio-app-rdm/tree/master/invenio_app_rdm/theme/assets/templates/search).
+
+## Change the record landing page
+
+When you click on a search result, you navigate in the details page of a specific record. We show how you can change the way the information is displayed in the details page.
```

Comment:
Moderate: On @fenekku comment above "We show you" --> We are going to change...? Like friendly, we do it all together but less bossy heh